src/components/Identity/constants.js

```diff
@@ -0,0 +1,12 @@
+const AUTH_STATES = {
```

Comment:
Will those be used anywhere other Identity? If so, we should move them with the other top level constants.

Reply:
I don't know of any other component using these states